internal/cache/metadata/type_cache_test.go

```diff
@@ -167,9 +168,6 @@ func (t *TypeCacheTest) TestGetAfterMaxEntriesExpiration() {
 
 	// The first inserted entry should have been evicted by all the later insertions.
 	ExpectEq(UnknownType, t.cache.Get(beforeExpiration, fmt.Sprint(0)))
-
-	// The second entry should not have been evicted
-	ExpectEq(RegularFileType, t.cache.Get(beforeExpiration, fmt.Sprint(1)))
```

Comment:
why did we remove this ?

Reply:
it was difficult to exact cut-off around which the the first entry would be evicted, but 2nd won't be. I have now standardized each entry's size by fixing the key string's size.